src/materialized/bin/materialized.rs

```diff
@@ -158,6 +165,10 @@ fn run() -> Result<(), failure::Error> {
     let processes = popts.opt_get_default("processes", 1)?;
     let address_file = popts.opt_str("address-file");
     let gather_metrics = !popts.opt_present("no-prometheus");
+    let bind_to = popts
+        .opt_str("bind-to")
+        .map(|s| SocketAddr::from_str(&s))
+        .transpose()?;
```

Comment:
This'll be cleaner with https://docs.rs/getopts/0.2.21/getopts/struct.Matches.html#method.opt_get.